File: all_of_python_and_c/conditionals/time.c
//Ryan Crop, jHow to get time in c
#include <stdio.h>
#include <time.h>


int main(void){
    //time_t seconds;
    //seconds = time(NULL);
    //printf("Seconds since jaunuary 1, 1970 = %d\n", seconds);

//Current time
    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    printf("Our current time and date is %s", asctime(timeinfo));

    //Gives us the current hour
    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("%d", hour);
    return 0;
}

File: all_of_python_and_c/conditionals/time.py
#Ryan Crop, How to get time of day

import time
# gmtime(0) is the first instense
#print(time.gmtime(0))

#curent time in secondes sense the first time 1970
#print(time.time())

#curre3nt time in seconds sense gmtime
current = time.time()

#Day date hour minute seconds year
now = time.ctime(current)
print(now)

#Get just the hour
local_time = time.localtime(current)
print(local_time)
hour = local_time.tm_hour
minute = local_time.tm_min
print(hour)